scripts/verify-replace.sh

```diff
@@ -39,6 +40,7 @@ do
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
+    echo ${API_REPLACE} >> go.mod
```

Comment:
I am no expert on `go.mod`, but how does this accomplish to add the api to the list of replaced modules?

Reply:
You need to think of ways this might break. e.g. what you might hold in API_REPLACE is gibberish, so a check with `go list -f` (or sth like this) is mandatory to ensure you are going to add a valid entry in go.mod. Or this line might already be there, so don't add it twice. I would prefer using Go tooling for this, that parses the file, rather than treating is a plain txt. Your solution would be fine if you have plain text files, but there you have a format, a valid lexer and parser. Adding things with echo in the very end, it's a no-no ;) I mean, it's good enough for testing your idea, but the implementation needs some love :)

Reply:
i have updated the logic now PTAL